apiml/src/main/resources/application.yml

```
@@ -127,6 +127,38 @@ apiml:
         headersToBeCleared: X-Certificate-Public,X-Certificate-DistinguishedName,X-Certificate-CommonName
         ssl:
             nonStrictVerifySslCertificatesOfServices: false
+        filterChainConfiguration: new
+        allowTokenRefresh: false
+        jwtInitializerTimeout: 5
+        personalAccessToken:
+            enabled: false
+        useInternalMapper: false
+        auth:
+            provider: zosmf
+            zosmf:
+                jwtAutoconfiguration: jwt
+        oidc:
+            enabled: false
+            clientId:
+            clientSecret:
+            registry:
+            identityMapperUrl:
+            identityMapperUser:
+            jwks:
+                uri:
+            userInfo:
+                uri:
+            validationType: JWK
+            provider: zosmf
+        x509:
+            enabled: false
+            certificatesUrl:
+            externalMapperUrl:
+        saf:
+            provider: rest
+            urls:
+                authenticate: https://${apiml.service.hostname}:10013/zss/saf/authenticate
+                verify: https://${apiml.service.hostname}:10013/zss/saf/verify
```

```suggestion
                authenticate: ${apiml.service.scheme}://${apiml.service.hostname}:${apiml.service.port}/zss/saf/authenticate
                verify: ${apiml.service.scheme}://${apiml.service.hostname}:${apiml.service.port}/zss/saf/verify
```

Why 10013 and not 10010?

---

just a copy paste from zaas. it is using zss port directly there too. we can change it, but it's not related to these changes

---

I wouldn't make this change right now without checking well the implications and effect.

---

ZSS is also almost unused in v3 (internal mapper is there, there's a native saf check implementation too) so it's likely these are not in use
I'd create an issue

---

https://github.com/zowe/api-layer/issues/4178